### ISSUE TITLE
🐞[bug]: Duplicate Navbar Displayed on cart.html

### ISSUE THREAD

**Issue by shashankgoud18** [Anjaliavv51/Retro#225]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

It appears that the navigation bar is being displayed twice on the cart.html page, as shown in the attached screenshot. One of these instances should be removed to improve the page layout.

### Expected behavior

The navigation bar should appear only once at the top of the page, similar to the other pages. The duplicate instance should be removed for a cleaner design and improved user experience.

### Add ScreenShots

![Screenshot 2024-10-05 190824](https://github.com/user-attachments/assets/96c14204-5426-4c95-a887-30ad1242a19d)

### What browsers are you seeing the problem on?

Microsoft Edge

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24EXTD contributor
- [X] I have starred the repository

**Comment by Anjaliavv51** [comment]
@shashankgoud18 it already assigned, sorry , create another issue.
